### PR TITLE
feat(menu-status): Add optional menu item 'status' property

### PR DESCRIPTION
**Pull request by ChefHutch**
**Code Changes**

- Move menu item SVGs out of the `svg/icons` directory (my bad for putting them in there)
- Add `LinkStatus` styled `Text` component to `MenuEntry.tsx`
- Imported this `LinkStatus` comp to the MenuEntry & Accordian components within `PanelBody` 
- Created `LIVE`, `SOON` & `NEW` status consts to pass to the menu config, these can be bypassed if necessary, so long as the LinkStatus object matches the interface:
- Created `LinkStatus` interface - it accepts a text string and a color string:
```js
 LinkStatus {
  text: string;
  color: keyof Colors;
}
```
- Added `status` objects to Storybook config for Farms, Team Battle & Profile & Teams > Leaderboard menu items.

**Notes** 
- Not sure if we want to add this to accordion items? Code [here](https://github.com/pancakeswap/pancake-toolkit/pull/29/files#diff-bc5716fdaf12b956865cb6fcdaa0cfb4114c5f0689f9ea4c66ba64c072643162R57-R62) I can easily rip that out. If we want to keep it - I wrapped the accordion label text within the `LinkLabel` component so it gets block styles

Screenshot 2021-03-24 at 14 24 26

### CODE REVIEW

File: packages/pancake-uikit/src/components/Svg/Icons/CircleOutline.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
```
Comment (ChefHutch) [inline packages/pancake-uikit/src/components/Svg/Icons/CircleOutline.tsx:1]: Being a sneaky bish and trying to sneak an unrelated (Competition page) svg into this PR 🙏

File: packages/pancake-uikit/src/widgets/Menu/components/MenuEntry.tsx [hunk 2]
```diff
@@ -66,6 +68,15 @@ MenuEntry.defaultProps = {
   role: "button",
 };
 
+const LinkStatus = styled(Text)<{ color: keyof Colors }>`
+  border-radius: 16px;
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/widgets/Menu/components/MenuEntry.tsx:72]: ${({ theme, color }) => theme.radii.default};

File: packages/pancake-uikit/src/widgets/Menu/statusConsts.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const LIVE = { text: "LIVE", color: "failure" };
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/widgets/Menu/statusConsts.ts:1]: this could go in the config file, with an object `Status`

**Review by RabbitDoge — CHANGES_REQUESTED** [review]

File: packages/pancake-uikit/src/widgets/Menu/statusConsts.ts (hunk 3 above)
Comment (ChefHutch) [reply packages/pancake-uikit/src/widgets/Menu/statusConsts.ts]: Added that in here, see if you're happy with the implementation? https://github.com/pancakeswap/pancake-toolkit/pull/29/files#diff-93fe9389e18f0ecc161575820214af1f993d1c81688a0f2a24d6c9bca4fb9628R1-R17

**Review by hachiojidev — APPROVED** [review]

File: packages/pancake-uikit/src/widgets/Menu/statusConsts.ts (hunk 3 above)
Comment (ChefHutch) [reply packages/pancake-uikit/src/widgets/Menu/statusConsts.ts]: Tweaked this and exported them. https://github.com/pancakeswap/pancake-toolkit/pull/29/files#diff-93fe9389e18f0ecc161575820214af1f993d1c81688a0f2a24d6c9bca4fb9628R3-R16

**Review by hachiojidev — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**